apps/mull-ui/src/app/components/event-page-info/event-page-info.tsx

```diff
@@ -17,11 +17,13 @@ import { ExpandableText } from './../expandable-text/expandable-text';
 import MullButton from '../mull-button/mull-button';
 
 export interface EventPageInfoProps {
-  event: IEvent;
+  event: Partial<IEvent>;
   className?: string;
+  /* This button will be used to create events in Event Review mode, and to join events in Event Preview mode, which is why the name is abstracted to handleMullButton */
+  handleMullButton?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
 }
 
-export const EventPageInfo = ({ event, className = '' }: EventPageInfoProps) => {
+export const EventPageInfo = ({ event, className = '', handleMullButton }: EventPageInfoProps) => {
```

Comment:
Hey @RGPosadas did you know you can document your component like so. This way when someone hovers over the component to figure out what `handleMullButton` does it will show them the text next to the param. This way we won't need the comment above as well
```suggestion
/**
 * @param handleMullButton Function called when button at bottom of event-page-info is pressed
 */
export const EventPageInfo = ({ event, className = '', handleMullButton }: EventPageInfoProps) => {
```

Reply:
Ooh I see, thank you for the fyi!